javascript/exercises/validateCreditCardSolution.js

```diff
@@ -0,0 +1,63 @@
+/*
+Here are the rules for a valid number:
+Number must be 16 digits, all of them must be numbers
+You must have at least two different numbers represented (all of the numbers cannot be the same)
+The final number must be even
+The sum of all the digits must be greater than 33
+*/
+
+var validateCreditCard = function(creditCardNum){
+  
+  //Remove dashes from creditCardNum string
+  var ccNumberNoDashes = '';
+  for (var h = 0; h < creditCardNum.length; h++) {
```

Comment:
Can we make these all `i`? I don't think we'll be using the variable name for anything other than single loops